utils/network.go

```diff
@@ -24,6 +30,34 @@ func (n Network) URL() string {
 		return "https://alpha4.starknet.io"
 	case MAINNET:
 		return "https://alpha-mainnet.starknet.io"
+	case GOERLI2:
+		return "https://alpha4.starknet.io"
+	case INTEGRATION:
+		return "https://external.integration.starknet.io"
+	default:
+		return ""
+	}
+}
+
+type Chain string
```

Comment:
Is this type necessary?

Reply:
The chains could as well just be a string but I felt it might be cool to different ChainIds from normal `strings`

Reply:
Is it possible for us to use the `Network` type exclusively? It seems to serve the same purpose as this `Chain` type.

Reply:
Okay. Made changes

Please take a look